Sources/TuistGenerator/Mappers/ModuleMapMapper.swift

```diff
@@ -2,6 +2,35 @@ import Foundation
 import TSCBasic
 import TuistCore
 import TuistGraph
+import TuistSupport
+
+enum ModuleMapMapperError: FatalError {
+    case invalidTargetDependency(sourceProject: AbsolutePath, sourceTarget: String, dependentTarget: String)
+    case invalidProjectTargetDependency(sourceProject: AbsolutePath, sourceTarget: String, dependentProject: AbsolutePath, dependentTarget: String)
+
+    /// Error type.
+    var type: ErrorType {
+        switch self {
+        case .invalidTargetDependency, .invalidProjectTargetDependency: return .abort
+        }
+    }
+
+    /// Error description.
+    var description: String {
+        switch self {
+        case let .invalidTargetDependency(sourceProject, sourceTarget, dependentTarget):
+            return """
+            Invalid target dependency for target \(sourceTarget) of project at path \(sourceProject.pathString): \
+            can't find target '\(dependentTarget)'
+            """
+        case let .invalidProjectTargetDependency(sourceProject, sourceTarget, dependentProject, dependentTarget):
```

Comment:
Similar comment here. If we can provide hints on why it happened, I'd add them. Otherwise, we can leave it as it is.